kaggle_cli/submit.py

```diff
@@ -35,7 +35,12 @@ def take_action(self, parsed_args):
         username = config.get('username', '')
         password = config.get('password', '')
         competition = config.get('competition', '')
-        zip = config.get('zip', False)
+        zip_flag = config.get('zip', 'no')
```

Comment:
You can use [getboolean](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getboolean) method instead.

Reply:
@floydwch but the `config` object here is just a python dict, it's not a ConfigParser object

Reply:
This issue should be resolved at the config provider level, I'll address it.